.github/workflows/ci.yml

```
@@ -42,4 +35,58 @@
-        run: pnpm test
+        run: pnpm test
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+  test-macos:
+    needs: quality           # optional
+    runs-on: macos-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)